examples/language-modeling/run_mlm_wwm.py

```diff
@@ -45,8 +45,13 @@
     set_seed,
 )
 from transformers.trainer_utils import is_main_process
+from transformers.utils import logging as hf_logging
 
 
+hf_logging.enable_default_handler()
+hf_logging.set_verbosity_info()
+hf_logging.enable_explicit_format()
+
```

Comment:
Don't think we need that here as it's already handled below cc @sgugger

Reply:
Where is-it handled in the file? I don't see it.

Reply:
Line 188

Reply:
Line 188 I have:
```
logger.warning(
        f"Process rank: {training_args.local_rank}, device: {training_args.device}, n_gpu: {training_args.n_gpu}"
        + f"distributed training: {bool(training_args.local_rank != -1)}, 16-bits training: {training_args.fp16}"
    )
```
Not related.